packit_service/worker/handlers/forges.py

```diff
@@ -90,3 +96,119 @@ def run(self) -> TaskResults:
 
         logger.info(msg)
         return TaskResults(success=True, details={"msg": msg})
+
+
+@reacts_to(event=IssueCommentEvent)
+class GithubFasVerificationHandler(JobHandler):
+    task_name = TaskName.github_fas_verification
+
+    def __init__(
+        self,
+        package_config: PackageConfig,
+        job_config: JobConfig,
+        event: dict,
+    ):
+        super().__init__(
+            package_config=package_config,
+            job_config=job_config,
+            event=event,
+        )
+        self.sender_login = self.data.actor
+        self.comment = self.data.event_dict.get("comment")
+        self._issue = None
+
+    @property
+    def issue(self):
+        if not self._issue:
+            self._issue = self.project.get_issue(self.data.issue_id)
+        return self._issue
+
+    def pre_check(self) -> bool:
+        """
+        Checks whether the Packit verification command is placed in
+        packit/notifications repository in the issue our service created.
+        """
+        if not (
+            self.project.namespace == "packit" and self.project.repo == "notifications"
+        ):
+            logger.debug(
+                "Packit verification comment command not placed in packit/notifications repository."
+            )
+            return False
+
+        issue_author = self.issue.author
+        if (
+            self.service_config.deployment == Deployment.prod
+            and issue_author != "packit-as-a-service[bot]"
+        ) or (
+            self.service_config.deployment == Deployment.stg
+            and issue_author != "packit-as-a-service-stg[bot]"
+        ):
+            logger.debug(
+                f"Packit verification comment command placed on issue with author "
+                f"other than our app: {issue_author}"
+            )
+            return False
+
+        return True
+
+    def run(self) -> TaskResults:
+        """
+        Discover information about organization/user which wants to verify the FAS account.
+        Allowlist automatically if mapping from github username to FAS account can prove a match.
+
+        Returns:
+            TaskResults
+        """
+        logger.debug(
+            f"Going to run verification of FAS account triggered by comment:"
+            f" {self.comment}"
+        )
+        # e.g. User Bebaabeni needs to be approved.
+        account_type, account_login, _ = self.issue.title.split(maxsplit=2)
+        namespace = f"github.com/{account_login}"
```

Comment:
nit: account_login can also be an organization, right?

Reply:
yes, therefore the `account_type` before which can be `organization` or `user`...or what am I missing? :D

Reply:
oh, right! I got confused about the naming of the variables, also 6pm on Fri - very little brain power :) thanks for explaining!